eng/emitter-package.json

```diff
@@ -1,11 +1,19 @@
 {
   "main": "dist/src/index.js",
   "dependencies": {
-    "@azure-tools/typespec-rust": "0.3.0"
+    "@azure-tools/typespec-rust": "0.5.1"
   },
   "devDependencies": {
-    "@azure-tools/typespec-client-generator-core": "0.48.4",
-    "@typespec/compiler": "0.62.0",
-    "@typespec/xml": "0.62.0"
+    "@azure-tools/typespec-autorest": "~0.49.0",
+    "@azure-tools/typespec-azure-core": "~0.49.0",
+    "@azure-tools/typespec-client-generator-core": "0.49.0",
```

Comment:
This will need to be bumped to `0.49.1` once emitter version `0.6.0` is picked up.